src/rust/engine/process_executor/src/main.rs

```diff
@@ -26,6 +27,13 @@ fn main() {
   env_logger::init();
 
   let args = App::new("process_executor")
+    .arg(
+      Arg::with_name("work-dir")
+        .long("work-dir")
+        .takes_value(true)
+        .required(true)
```

Comment:
Let's remove the `.required(true)`, and default this to `std::env::temp_dir()` by on line 106 doing a `.or_else(std::env::temp_dir)` call instead of `.unwrap()`

Reply:
done